source/_docs/logs.md

```diff
@@ -112,7 +112,41 @@ You now have a local copy of the logs directory, which contains the following:
     └──mysqld-slow-query.log
     └──mysqld.log
 ```
-To automate this process, see [Automate Downloading Logs from the Live Environment](/docs/download-logs/).
+
+## Automate Downloading Logs
+
+You can automate the process of accessing and maintaining these logs with a simple script.
+
+### Enable Passwordless Access
```

Comment:
Recommend enabling passwordless access regardless of download method - maybe move this section to under the available logs table?